modules/01-mods.js

```diff
@@ -243,18 +237,22 @@ async function slashModMute(interaction) {
         + 'A member of the mod team will be available to discuss more details.\n\n'
         + 'http://ldsgamers.com/code-of-conduct'
       );
-    } else { // Remove mute
+
       await interaction.editReply({
-        content: `Unmuting ${target}...`,
-        ephemeral: true
+        content: `Muted ${target}.`,
       });
-
+    } else { // Remove mute
       // Don't unmute if not muted
-      if (!target.roles.cache.has(sf.roles.muted)) return;
+      if (!target.roles.cache.has(sf.roles.muted)) {
+        await interaction.editReply({
+          content: `${target} isn't muted.`,
+        });
+        return;
+      }
 
       // Remove Mute
       await target.roles.remove(sf.roles.muted);
-      if (muteState.get(target.id)) await target.voice.setMute(false, "Mute resolved");
+      if (muteState.get(target.id) && target.voice.channel) await target.voice.setMute(false, "Mute resolved");
```

Comment:
This doesn't seem right... If `muteState` is to hold onto the user's mute status _before_ the mute so that we can restore it to whatever it was before, shouldn't this be the following? I think we need to check with Gai for his intent with `muteState`.
```suggestion
      if (!muteState.get(target.id) && target.voice.channel) await target.voice.setMute(false, "Mute resolved");```

Reply:
You're correct, I believe. I misunderstood it